ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -188,6 +188,7 @@ statsByClause
    : BY fieldList
    | BY bySpanClause
    | BY bySpanClause COMMA fieldList
+   | BY fieldList COMMA bySpanClause
```

Comment:
Is span expression mixed in field list also valid case?

Reply:
From the changes it looks we are missing that.
```
: (fieldExpression | bySpanClause) (COMMA (fieldExpression | bySpanClause))*
```

Something like this would actually be a good idea or definining a new fieldList.

Reply:
yeah, if this is valid, probably we should treat span expression as regular expression and do other validation outside ANTLR. Will approve now. Please check if we need to change this in future. Thanks!

Reply:
> probably we should treat span expression as regular expression 

The reason separating Span from field list AFAIK because that grouping by more than one time-span seems meaningless. And there is one span expression in Aggregation currently: https://github.com/opensearch-project/sql/blob/b2403ca4fa1bbba2a1ac8827b6f7aeefe48f9f32/core/src/main/java/org/opensearch/sql/ast/tree/Aggregation.java#L28

In general, it doesn't make sense that grouping by a 10-minutes span and 5-minutes span together.
But I tried in SPL, query group by two time span could be executed:
```
sourcetype=access_* status=200 action=purchase | bin span=10m _time as 10m_span | bin span=5m _time as 5m_span | stats count by 10m_span, 5m_span, clientip
```
Yes, we could refactor in future.